src/mmw/js/src/draw/controllers.js

```diff
@@ -51,6 +55,29 @@ var DrawController = {
     }
 };
 
+/**
+ * If we are in embed mode then the project is an activity and we want to keep
+ * the same project reguardless of changes to the AOI. This prepares a project
+ * immedialty upon visiting the page and will be the only project the user can
+ * save during this session.
+ */
+function enableSingleProjectModeIfActivity() {
+    if (settings.get('activityMode')) {
+        if (!App.currProject) {
+            var project = new modelingModels.ProjectModel({
+                name: 'New Project',
```

Comment:
Originally I had this named in a way that would differentiate it from the normal projects. This is generic enough that I should either make it more specific such as "New Activity" or just make it the same "Untitled Project" we are using elsewhere.